### ISSUE TITLE
Can't copy content from Warpified subshell block

### ISSUE THREAD

**Issue by zheng** [warpdotdev/Warp#4869]
### Dupe Check

- [X] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates

### Describe the bug

I'm unable to copy any content from the warpified shell block. The same problem applies to "insert into input"

### To reproduce

1. zsh
2. Ctrl + I
3. Wait for block to get warpified
4. Try to copy text from this block

### Operating system

MacOS

### Operating system and version

Sonoma 14.3

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by dannyneira** [comment]
Thanks @zheng ❤️ 

I was able to reproduce this with the first block, but not any subsequent ones, do you see this when running any commands on a warpified subshell?

**Comment by zheng** [comment]
Yes, it's only with the "Success! This subshell has been warpified" block specifically :D. The reason why I care about it, is it has the config line to automatically warpify. Without being able to copy it, I needed to type it out manually

**Comment by dannyneira** [comment]
I see, thats less than ideal, In the meantime we also have them in our docs and I'll post them here for anyone else may need them. https://docs.warp.dev/features/subshells#automatically-warpify-subshells

```
# For zsh subshells, add to ~/.zshrc.
printf '\eP$f{"hook": "SourcedRcFileForWarp", "value": { "shell": "zsh"}}\x9c'

# For bash subshells, add to ~/.bashrc or ~/.bash_profile.
printf '\eP$f{"hook": "SourcedRcFileForWarp", "value": { "shell": "bash"}}\x9c'

# For fish subshells, add to ~/.config/fish/config.fish.
if status is-interactive
  printf '\eP$f{"hook": "SourcedRcFileForWarp", "value": { "shell": "fish"}}\x9c'
end
```
